app/org/maproulette/framework/service/NotificationService.scala

```diff
@@ -192,13 +192,34 @@ class NotificationService @Inject() (
     // match [@username] (username may contain spaces) or @username (no spaces allowed)
     val mentionRegex = """\[@([^\]]+)\]|@([\w\d_-]+)""".r.unanchored
 
+    // Challenge owners should be notified everytime a challenge comment is posted unless its their comment
+    if (fromUser.id != challenge.general.owner) {
+      this.addNotification(
+        UserNotification(
+          -1,
+          userId = challenge.general.owner,
+          notificationType = UserNotification.NOTIFICATION_TYPE_CHALLENGE_COMMENT,
+          fromUsername = Some(fromUser.osmProfile.displayName),
+          taskId = None,
+          challengeId = Some(challenge.id),
+          targetId = Some(comment.id),
+          extra = Some(comment.comment)
```

Comment:
What's the end result of this comment in the UI? I ask because if this is a HUGE comment, like a few hundred words, does it make sense to do something like keep only the first 100 or so chars and end it with "..."? Another impact is avoiding sending out a large amount of data unnecessarily.

Reply:
I'll discuss message limitations with @mvexel.  this is a good thing to bring up but should be issued separately from this feature and incorporated globally.